### DISCUSSION TITLE
MMT Resources to continue learning

### DISCUSSION THREAD

**Discussion by sheridanvk** [sheridanvk/anything-we-can-do#28] [Modern Monetary Theory Resources]
Welcome to the MMT Discussion board! Here's some ideas of resources to help you become even more of an expert in Modern Monetary Theory and alternative ways of building our economy. If there's others you like that you don't see here, add them to the comment thread or create your own Discussion Topic.

## Books and other readings
- [The Deficit Myth](https://www.johnmurraypress.co.uk/titles/stephanie-kelton-2/the-deficit-myth/9781529352566/) - an accessible overview of Modern Monetary Theory, which goes into detail about how governments currently manage their economies, and why fears of "the national debt" is unnecessarily holding them back.
- [Less is More](https://uk.bookshop.org/p/books/less-is-more-how-degrowth-will-save-the-world-jason-hickel/364774) - an overview of how our economies came to be driven by growth, and why it's so disastrous for our planet and most people living on it. It presents a compelling vision for how we could be doing things differently
- [Doughnut Economics](https://uk.bookshop.org/p/books/doughnut-economics-seven-ways-to-think-like-a-21st-century-economist-kate-raworth/2694262) - an overview of how so much conventional wisdom on the economy is just totally wrong, and how we should be thinking about making sure we provide for everyone's basic needs, without going past planetary boundaries
- [How to Pay for Saving the World](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4172005) - this accessible academic paper is a fantastic overview of how we can completely change our economy to get back within planetary boundaries by using insights from Modern Monetary Theory
- [David Graeber's Debt](https://uk.bookshop.org/p/books/debt-the-first-5000-years-david-graeber/2823513) - this book is long, but it's an incredible, well-written history of the last 5000 years of money, debt and society.
- [Randy Wray's MMT Primer](https://realprogressives.org/mmt-primer/) - very in-depth, fantastic deep dive into all things MMT

## Podcasts and Videos
- [Finding the Money](https://findingmoneyfilm.com) - feature-length film describing how money works in clear and entertaining detail. 
- [The MMT Podcast](https://pileusmmt.libsyn.com/) - an amazing treasure trove of detail on how all of this works in detail. Start with the first 3 episodes for an overview of MMT, and then explore the archive for anything that interests you - I very much recommend this episode on [how banks create money](https://pileusmmt.libsyn.com/13-steven-hail-everything-you-always-wanted-to-know-about-banking-but-were-afraid-to-ask), and this one on [whether the markets have too much influence on our politics](https://pileusmmt.libsyn.com/147-dirk-ehnts-do-markets-control-our-politics) (spoiler: only if we let them!)
  - Another great podcast (US-based but with many international guests) is [Macro n Cheese]([realprogressives.org/macro-n-cheese-podcast-quick-ref-list](https://realprogressives.org/macro-n-cheese-podcast-quick-ref-list/))
- [Planet Critical](https://www.planetcritical.com/podcast) - wide-ranging focus on many causes of the crises we're facing. There's also [a fantastic episode focused on MMT](https://www.planetcritical.com/p/what-we-get-wrong-about-money).
- [Modern Money Lab](https://www.youtube.com/channel/UC6jgosUfqfCv5fQtNdS422Q) has a fantastic range of videos on many topics - I particularly recommend Randy Wray's [The History and Nature of Money](https://www.youtube.com/watch?v=7cLDFjTt4Bs)

## Education and Conferences
- [Beyond Growth conferences in Europe](https://weall.org/beyond-growth-conferences-happening-across-europe)
- [Scotonomics](https://scotonomics.scot/live-events/) - podcasts and events focusing on how MMT insights can serve Scotland
- [Gower Institute for Modern Monetary Theory](https://gimms.org.uk) - many MMT resources specific to the UK, including their [inaugural MMT Conference](https://gimms.org.uk/2024/01/25/uk-mmt-conference/) this year at the University of Leeds, July 15-17
- [Free European MMT Conference and Summer School](https://www.mmtconference.eu/uber-uns/) - dates to be announced for 2024
- [Graduate programmes in the Economics of Sustainability](https://www.torrens.edu.au/courses/business/graduate-certificate-of-economics-of-sustainability) - if you decide to get deep into this, there are online graduate courses available through Torrens University, Australia (graduate certificate, graduate diploma or master's programme available)

**Comment by Joules1971** [comment]
Great list!

Macro N Cheese podcast has a lot of great episodes too. It’s US based but there are lots of international guests. https://realprogressives.org/macro-n-cheese-podcast-quick-ref-list/

I’d also recommend the Modern Money Primer from Randy Wray. https://realprogressives.org/mmt-primer/

**Reply by sheridanvk** [reply]
@Joules1971 thanks! Those are great suggestions; I'll add them to the list above :)
